py/server/deephaven/table_factory.py

```diff
@@ -396,7 +398,7 @@ def function_generated_table(table_generator: Callable[[], Table],
 
     def table_generator_function():
         with exec_ctx:
-            result = table_generator()
+            result = table_generator(**kwargs)
```

Comment:
The function call here could raise an exception of argument related error, such as unexpected argument etc. What should be our strategy to handle that? We probably should have some test cases to verify that the users won't be confused by the exceptions.

Reply:
It seems that the precedent here is to assume that the table-generating function is going to be responsible and correct. We already don't try to intercept any errors from that function, don't do any checks for its correctness, etc. I think it's reasonable to let the function complain about wrong arguments if it's passed wrong arguments, and leave the  responsibility to the user. I agree that we need unit tests to make sure that the exceptions raised are the "wrong argument" exceptions that I expect.